lib/ddtrace/tracer.rb

```diff
@@ -63,61 +63,16 @@ def call_context(key = nil)
     # * +enabled+: set if the tracer submits or not spans to the local agent. It's enabled
     #   by default.
     def initialize(options = {})
```

Comment:
Minor: Since we're massively changing this method, perhaps a good time to move to real kw args? :)

Reply:
I didn't want to, to not disturb Ruby 3.0 users, but I'm easily convinced. Made this change!